src/form.js

```diff
@@ -185,6 +185,12 @@ define([
         removeHashtag: function(hashtag) {
             delete this.hashtagDictionary[hashtag];
         },
+        clearNullHashtags: function () {
+            this.hashtagDictionary = _.chain(this.hashtagDictionary)
```

Comment:
https://lodash.com/docs#omitBy

Was not trivial to migrate teh last time I looked at it, but that was when we were on 1.4

https://github.com/lodash/lodash/wiki/Migrating that's probably from 1.8 to lodash. not sure how feasible it is to migrate to 1.8, but would be happy if we did :)